src/scripts/lychee.js

```diff
@@ -144,6 +144,12 @@ lychee.logout = function() {
 
 }
 
+lychee.restart = function() {
+
+	document.location.href = ''
```

Comment:
That reloads the URL including '#...'. As I just tested, that works as well, because we get redirected to the URL without '#...' afterwards. Not sure why.